src/components/ArtCard/ArtCard.css

```diff
@@ -0,0 +1,26 @@
+/* OVERRIDE SEMANTIC DEFAULT */
+.ui.card>:last-child,
+.ui.cards>.card>:last-child {
```

Comment:
tell me about this syntax later bc this is not something i know but would like to